ksqldb-engine/src/main/java/io/confluent/ksql/query/MissingTopicClassifier.java

```diff
@@ -48,7 +48,12 @@ public MissingTopicClassifier(
 
   @Override
   public Type classify(final Throwable e) {
-    LOG.info("Attempting to classify error for {}", queryId);
+    LOG.info(
```

Comment:
added this to this PR because in a recent incident I thought this would be nice to have, unrelated to the main change